### PR TITLE
Improved UX of executing oscap right after compilation.

### PR DESCRIPTION
**Pull request by matejak**
* ~Renamed run to oscap_wrapper.~
* ~Make the compiled oscap invocation part of the scipt.~
* Introduced the oscap_wrapper script that contains the compiled oscap invocation.
* Improved comments of the run script, to make it's usage more explicit.

Making this 1.3-only as this heavily depends on the build system.

### CODE REVIEW

**Comment by jan-cerny** [comment]
@matejak 
You are great!


This has to be changed also in `tests/test_common.sh.in`. 

Is `oscap_wrapper` the best name? What about keeping `run` or just `run_oscap`?

Also, I think it would be great to document it in Developer Guide.

**Comment by evgenyz** [comment]
@jan-cerny It was `run` when it was able to execute arbitrary stuff, passed in the args. Now it's for oscap only. And I personally would recommend to name a wrapper by adding suffixes rather than prefixes (muscle memory, autocomplete etc.).

@matejak Is it okay that this wrapper is becoming oscap-only? Are there no other things to run?

**Comment by jan-cerny** [comment]
@evgenyz I have never used this wrapper for anything else than `oscap`. I think it is OK to make it dependent. Have you used it together with something else than `oscap`?

**Comment by evgenyz** [comment]
@jan-cerny I never used it yet even for oscap invocation, actually :). I'm just curious why it was parametrized in the first place.

**Comment by jan-cerny** [comment]
@matejak @evgenyz I have realized that this change will prevent us from running `valgrind`, `gdb`, etc. on `oscap`.

**Comment by matejak** [comment]
I have rewritten the improvement to get the best from both worlds.

**Comment by evgenyz** [comment]
LGTM

File: oscap_wrapper.in [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -
+# openscap 'run' programs locally script
+# Copyright (C) 2011-2013 Red Hat Inc.
```
Comment (jan-cerny) [inline oscap_wrapper.in]: 2019

File: oscap_wrapper.in [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+#----------------------------------------------------------------------
+
+# With this script you can run oscap without needing to
+# install the utility first.  You just have to do for example:
+#
+#   ./oscap_wrapper [oscap args ...]
+#
+# This works for any C program or test that uses the openscap
+# libraries.
```
Comment (jan-cerny) [inline oscap_wrapper.in]: it doesn't work for any program, it wraps oscap

**Comment by matejak** [comment]
Nice thorough observation, fixed!

**Pull request merged**